src/main.c

```diff
@@ -632,6 +632,18 @@ void vb_statusbar_update(Client *c)
 
     statusbar_update_downloads(c, status);
 
+    if ( c->config.statusbar_show_settings ) {
+        /* show js, cookies, and local storage status */
+
+        Setting *js = g_hash_table_lookup(c->config.settings, "scripts");
+        Setting *cookies = g_hash_table_lookup(c->config.settings, "cookie-accept");
+        Setting *local_storage = g_hash_table_lookup(c->config.settings, "html5-local-storage");
```

Comment:
@Sadoon-AlBader Could you please use the macros out of src/main.h to read the config settings. `GET_BOOL(c, "scripts")`, `GET_CHAR(c, "cookie-accept")` and `GET_BOOL(c, "html5-local-storage")`

Reply:
I wish I had seen those before, wasted a lot of time trying to come up with my own way xD

Reply:
Alright, done. Seems to work well.